Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# New lifestyles (second master branch)

#### Changelog:
- Added Magic Lifestyles.
- Added effects to sorcerer traits
- Changed localisation for magical doctrine parameters
- Close #5

#### How to test:
- Basically, it's 2nd round of what have been tested in #120 - check that lore character can pick correct focuses in correct lifestyles.
- Ignore perks, they are not ready yet.
- Use `add_lifestyle_xp_all` console command to get exp for all lifestyles
